.github/workflows/deploy.yml

```diff
@@ -9,14 +9,18 @@ jobs:
     runs-on: ubuntu-latest
     steps:
     - uses: actions/checkout@v1
-    - uses: actions/setup-node@v1
+    - name: Read .nvmrc
+      run: echo ::set-output name=NVMRC::$(cat .nvmrc)
+      id: nvm
+    - name: Setup node
+      uses: actions/setup-node@v1
       with:
-        node-version: '12.x'
-    - run: yarn install
+        node-version: '${{ steps.nvm.outputs.NVMRC }}'
+    - run: npm install
     - name: Deploy the extension
       run: |
-        yarn run grunt publish:chrome
-        yarn run grunt publish:firefox
+        npx run grunt publish:chrome
+        npx run grunt publish:firefox
```

Comment:
`npx <package-name>`, not `npx run <package-name>`.

Reply:
Will verify locally